### PR TITLE
chore: Project addition by Manoj-Kumar-Munda #2164

### LINKED ISSUE #2164: chore: project addition by
### Add a new project to the list

I want to add my new frontend project which is a food delivery website. 

### Record

- [X] I have checked the existing [issues](https://github.com/priyankarpal/ProjectsHut/issues)
- [X] I have read the [Contributing Guidelines](https://github.com/priyankarpal/ProjectsHut/blob/main/contributing.md)
- [X] I agree to follow this project's [Code of Conduct](https://github.com/priyankarpal/ProjectsHut/blob/main/CODE_OF_CONDUCT.md)
- [X] I want to work on this issue

### PR DESCRIPTION
**Pull request by Manoj-Kumar-Munda**
## Related Issue

closes #2164

## Description
Added a new project to DB/project.json

## Screenshots
Screenshot (10)

### CODE REVIEW

File: DB/projects.json [hunk 1]
```diff
@@ -4755,5 +4755,24 @@
...
+    "github_username": "Manoj-Kumar-Munda",
+    "Projects": [
+      {
+        "github_url": "https://github.com/Manoj-Kumar-Munda/CraveBite",
+        "title": "CraveBite",
+        "description": "A food ordering web application",
+        "tech": [
+          "html",
+          "css",
+          "tailwind css",
```
Comment (priyankarpal) [inline DB/projects.json:4769]:
```suggestion
          "tailwindcss",
```

File: DB/projects.json [hunk 2]
```diff
@@ -4755,5 +4755,24 @@
...
+          "reactjs",
+          "javascript",
+          "redux toolkit",
```
Comment (priyankarpal) [inline DB/projects.json:4772]:
```suggestion
          "reduxtoolkit",
```

**Review by priyankarpal — CHANGES_REQUESTED** [review]

File: DB/projects.json [hunk 3]
```diff
@@ -4768,7 +4768,7 @@
           "tailwindcss",
           "reactjs",
           "javascript",
-          "redux toolkit",
+          "reduxtoolkit",
```
Comment (Manoj-Kumar-Munda) [inline DB/projects.json:4771]: changed the text "redux toolkit" to "reduxtoolkit"

File: DB/projects.json [hunk 4]
```diff
@@ -4765,8 +4765,7 @@
         "description": "A food ordering web application",
         "tech": [
           "html",
-          "css",
-          "tailwind css",
+          "tailwindcss",
```
Comment (Manoj-Kumar-Munda) [inline DB/projects.json:4768]: removed "css" and "tailwind css" and added "tailwindcss" in tech

**Review by Manoj-Kumar-Munda — COMMENTED** [review]
made changes in the tech items

**Pull request closed without merging**